README.md

```diff
@@ -9,4 +9,158 @@ To install this project, you can simply run `sh install.sh` in your terminal. Th
 To run the scraper, you can use the run script by typing `sh run_scraper.sh` into your terminal. This will enable the virtual environment and run `scraper.py`. Once again, the virtual environment will not stay active after the script finishes running. If you want to run the scraper without the run script, enable the virtual environment, then run `python3 scraper.py`.
 
 ## Running the API
-The best way to run the API right now is to run the command `FLASK_APP="app.py" FLASK_ENV=development flask run;`.
+The best way to run the API right now is to run the command `FLASK_APP="app.py" FLASK_ENV=development flask run;`. Note that this is not the best way to run the scraper at this time.
+
+## Data Model
+The following sections document the differences between the counties in the common data model (see `data_models` directory) which we will see as we begin to get data from them.
+
+### Ages
+
+Please make sure to use the following age brackets for the different counties. Note that the brackets may also vary by whether you are scraping cases or deaths data:
+
+
+#### San Francisco
+##### Cases
+	"age": [
+    	{"group": "18_and_under", "raw_count": -1 },
```

Comment:
We should clean up the mixed tabs and spaces in all these examples before merging.

Reply:
It’s not just these two lines; there are mixed indents all throughout these.